File: src/main/java/ru/dragonestia/apocalypse/item/TinIngotItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class TinIngotItem extends Item implements AdvItem {

    public TinIngotItem(Integer meta, int count) {
        super(ApocalypseID.TIN_INGOT, meta, count, "Tin ingot");
    }

    @Override
    public String getRuName() {
        return "Оловянный слиток";
    }

    @Override
    public String getIcon() {
        return "textures/items/ingot/tin";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/TinIngotItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class TinIngotItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.TIN_INGOT, 0, count, "Tin ingot"), AdvItem {

    override fun getRuName(): String = "Оловянный слиток"

    override fun getIcon(): String = "textures/items/ingot/tin"

}